package/MDAnalysis/analysis/pca.py

```diff
@@ -561,7 +573,7 @@ def project_single_frame(self, components=None, group=None, anchor=None):
             for res in group.residues:
                 # n_common is the number of pca atoms in a residue
                 n_common = pca_res_counts[np.where(
-                           pca_res_indices == res.resindex)][0]
+            		   pca_res_indices == res.resindex)][0]
```

Comment:
This line contains a tab space, could you change this to only be standard spaces please?

Reply:
How did you notice?!

But yes, no TABS!!!